cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -316,9 +329,10 @@
         ["spike-in", False, "ERCC-00002 (spike-in control)", "NCBITaxon:32630", 1061, "synthetic"],
         ["gene", False, "MACF1", "NCBITaxon:9606", 2821, "protein_coding"],
         ["gene", False, "Trp53", "NCBITaxon:10090", 1797, "protein_coding"],
-        ["gene", False, "S", "NCBITaxon:2697049", 3822, "protein_coding"],
+        ["gene", False, "S_ENSSASG00005000004", "NCBITaxon:2697049", 3822, "protein_coding"],
```

Comment:
Why did feature_name change for this one if we didn't bump the version?

Reply:
so here: https://github.com/chanzuckerberg/single-cell-curation/blob/main/cellxgene_schema_cli/scripts/gene_processing.py#L211-L224

we deduplicate gene names across species. my guess is that fruit fly has a gene named `S`, which resulted in this being picked up and renamed to `[gene name]_[gene id]` instead of just `gene name`.